userdocs/userdocs.docc/debugging.md

```diff
@@ -0,0 +1,11 @@
+# Debugging
+
+vscode-swift allows you to debug your Swift packages.
+
+> Tip: Debugging works best when using a version of the Swift toolchain 6.0 or higher
+
+> Tip: Debugging uses workflows common to all VSCode extensions. For more information see https://code.visualstudio.com/docs/editor/debugging
```

Comment:
Two `> Tip:` in a row makes a pretty large footprint on the rendered page, and pushes the content just below way down. The second one of these I'm not even sure really rises to what I'd think of as a tip - it doesn't optimize anything or show you a smoother path - so I'd suggest moving the content about the fact that debugging uses the common VSCode extension workflow pieces to the end of this content, and use that as a wrapping up bit that provide a place to read for more general information.

(If there's something specific about the general workflows that's important to know when wanting to use debugging through VSCode, definitely call that out early - I'm guessing that this isn't the case here, and it's more of a "for more information..." bit for the curious)